### ISSUE TITLE
Adding a Dockerfile?

### ISSUE THREAD

**Issue by DanielLenz** [healpy/healpy#494]
Due to the current issues with OS X builds, I've been doing all the development/testing in a docker container.

Do you think it would add value to have a light-weight [Dockerfile](https://gist.github.com/DanielLenz/868e7544ccbea4fd255be6bb94238f50) as part of the healpy repo? This helps to overcome any installation/dependency issues, allows for easy development, and could e.g. be used to set up an interactive [binder](https://mybinder.org).

Developing/running/testing is then as easy as:

```bash

docker run -d -t -v "$(pwd)":/home/healpy hp:latest
docker exec -it <container_id> /bin/bash
```

**Comment by zonca** [comment]
yes, I think it would be very useful to have, and also have an autobuild on dockerhub

**Comment by DanielLenz** [comment]
While this has been merged, I'll keep it open to see if everything (especially the autobuild) works as intended.

**Comment by DanielLenz** [comment]
Not yet; let me look into the autobuild on dockerhub one more time first.

**Comment by lpsinger** [comment]
Daniel, what are the macOS issues? My main machine is a Mac; I can help.

**Comment by DanielLenz** [comment]
I just tried it out again (it's been a couple of months), and the latest (1.12.10) version installs just fine from source on MacOS with gcc-8. Thanks for offering your help, I'm sure I'll take you up on that offer sooner or later :)
